addons/web_dashboard/models/res_users.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+from openerp import models, api, _
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    def web_dashboard_get_users_by_state(self):
+        users = self.search([('login_date', '=', False)], order="create_date desc")
+        pending = users.filtered(lambda u: u.signup_valid)
+        expired = users - pending
+        return {'expired': zip(expired.mapped('id'), expired.mapped('login')), 'pending': zip(pending.mapped('id'), pending.mapped('login'))}
+
+    @api.model
+    def web_dashboard_create_users(self, email_adderesses, optional_message=""):
```

Comment:
Please read what you type: adderesse does not exists in English. Maybe you want to say adresse like in French, but I don't think so, actually :) .